### PR TITLE
DROOLS-2653: [DMN Designer] Replace 'My name' (etc) with better defaults

### PR DESCRIPTION
**Pull request by manstis**
See https://issues.jboss.org/browse/DROOLS-2653

### CODE REVIEW

**Comment by manstis** [comment]
@jomarko @tarilabs FYI, see https://youtu.be/SsuMgbMiaH4

File: kie-wb-common-dmn/kie-wb-common-dmn-client/src/main/java/org/kie/workbench/common/dmn/client/property/dmn/DefaultValueUtilities.java [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+        } else if (dmnModel instanceof Decision) {
+            updateDecisionDefaultName(graph, (Decision) dmnModel);
+        } else if (dmnModel instanceof InputData) {
+            updateInputDataDefaultName(graph, (InputData) dmnModel);
+        } else if (dmnModel instanceof KnowledgeSource) {
+            updateKnowledgeSourceDefaultName(graph, (KnowledgeSource) dmnModel);
+        } else if (dmnModel instanceof TextAnnotation) {
+            updateTextAnnotationDefaultText(graph, (TextAnnotation) dmnModel);
+        } else {
+            throw new IllegalArgumentException("Default Name for '" + dmnModel.getClass().getSimpleName() + "' is not support.");
```
Comment (jomarko) [inline kie-wb-common-dmn/kie-wb-common-dmn-client/src/main/java/org/kie/workbench/common/dmn/client/property/dmn/DefaultValueUtilities.java]: Should be the error message **.... is not supported.** ?

Comment (manstis) [reply kie-wb-common-dmn/kie-wb-common-dmn-client/src/main/java/org/kie/workbench/common/dmn/client/property/dmn/DefaultValueUtilities.java]: Yes it should! Your English is brilliant.

**Comment by karreiro** [comment]
Jenkins please retest this.

**Comment by manstis** [comment]
@jomarko I have merged your codebase reduction and resolved the inconsistency between "graph node names" and "expression element names". Both now work the same; being a lookup for a next unused index value (rather than node count). So, given a node "abc-5" the next will be "abc-6".

File: kie-wb-common-dmn/kie-wb-common-dmn-client/src/main/java/org/kie/workbench/common/dmn/client/property/dmn/DefaultValueUtilities.java (hunk 1 above)
Comment (jomarko) [reply kie-wb-common-dmn/kie-wb-common-dmn-client/src/main/java/org/kie/workbench/common/dmn/client/property/dmn/DefaultValueUtilities.java]: No it is not and we know it :)

**Comment by manstis** [comment]
@karreiro Could you please review?

**Comment by manstis** [comment]
Thanks @karreiro logged proposal [here](https://issues.jboss.org/browse/DROOLS-2697).

**Pull request merged**